app/src/main/java/com/sdp/movemeet/Sport.java

```diff
@@ -16,6 +16,6 @@
     Hockey,
```

Comment:
Didn't you add other additional sports (like yoga, trekking, etc.?). Why aren't those sports appearing here?

Reply:
By the way, if I give you the icons (you can find them here: https://we.tl/t-P9yH8jEUN2), do you think you could add those two sports as well:
- Parkour
- Tricking
This is for a personal later use ^^' ahah thanks in advance for the modifications if you have time!

Reply:
I think they don't appear in the list of changes because someone added them to the main (I think it was JL), as a way to prevent bugs when creating activities. But, if I don't mistaken, he didn't put the icons associated with the new sports.
Yep, I can add them, no problem ;)

Reply:
1) Ok I see 👍
2) Awesome, thank you so much! 😍 Ahah 😂